src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs

```diff
@@ -24,5 +24,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```

Comment:
Please don't update the `AssemblyVersion`

Reply:
Reverted.